bots/example.py

```diff
@@ -0,0 +1,47 @@
+import os
+from datetime import timedelta
+
+from ape_ethereum import multicall
```

Comment:
not technically used yet, I had this in some of my other code for making claims I didn't publish yet